packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { Icon } from "@cloudflare/component-icon";
+import { A, Span } from "@cloudflare/elements";
+import { isDarkMode } from "@cloudflare/style-const";
+import { createStyledComponent } from "@cloudflare/style-container";
+import React from "react";
+
+const Link = createStyledComponent(
+	({ theme }) => ({
+		display: "inline-flex",
+		alignItems: "center",
+		width: "fit-content",
+		fontSize: theme.fontSizes[1],
+		color: `${theme.colors.indigo[isDarkMode() ? 9 : 2]} !important`,
+		backgroundColor: theme.colors.indigo[isDarkMode() ? 4 : 9],
+		borderRadius: "100vw",
+		px: 2,
+		py: "1px",
+		textDecoration: "none",
+		lineHeight: 1.5,
+		verticalAlign: "bottom",
+		gap: 2,
+		"& svg": {
+			color: theme.colors.indigo[isDarkMode() ? 9 : 4],
+		},
+		"&:hover svg": {
+			color: theme.colors.indigo[isDarkMode() ? 9 : 2],
+		},
+	}),
```

Comment:
I feel like these styles should be shared somehow? Is this a button that looks similar to other buttons on the site, or are these styles specific to the docs link?

Reply:
The styling of this button is pretty unique to the docs link—it doesn't really look like anything else on the site